### PR TITLE
Fix transpose farm answers in check answers page

### PR DESCRIPTION
**Pull request by Cruikshanks**
https://eaflood.atlassian.net/browse/RUBY-99

An issue with the current version of the service is that the answers to whether the waste activity will

- take place on a farm
- the person responsible is a farmer

get transposed in the check your answers screen. When we migrated the functionality to this engine, we also inadvertently copied this same transposing issue.

The root cause is the bad naming of the routes. Prior to this change they were

- `is_a_farm` meant is the activity taking place on a farm
- `on_a_farm` meant is the person responsible a farmer

It's easy to see why with names like this the information gets transposed. So this change resolves the root problem by renaming the routes, and all related objects (controllers, forms, views, column names etc), and ensures the correct info is displayed with the correct title in the check your answers page.

### CODE REVIEW

**Comment by Cruikshanks** [comment]
I know CodeClimate is giving me the slap down, but I thought it was cleaner to leave the specs out whilst @eminnett is doing his 'thang'. At the very least total coverage remains unaffected 😬 .

File: app/views/waste_exemptions_engine/on_a_farm_forms/new.html.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <% else %>
     <div class="form-group">
     <% end %>
-      <fieldset class="inline" id="on_a_farm">
+      <fieldset class="inline" id="is_a_farm">
```
Comment (irisfaraway) [inline app/views/waste_exemptions_engine/on_a_farm_forms/new.html.erb]: Should this have been changed?

File: config/locales/forms/check_your_answers_forms/en.yml [hunk 2]
```diff
@@ -43,12 +43,12 @@ en:
...
         farming:
           subheading: Farming information
-          is_a_farm:
-            true_html: "the operater <span class=\"strong\">is</span> a farmer"
-            false_html: "the operater <span class=\"strong\">is not</span> a farmer"
           on_a_farm:
             true_html: "the operation <span class=\"strong\">will</span> take place on a farm"
             false_html: "the operation <span class=\"strong\">will not</span> take place on a farm"
+          is_a_farmer:
+            true_html: "the operater <span class=\"strong\">is</span> a farmer"
```
Comment (irisfaraway) [inline config/locales/forms/check_your_answers_forms/en.yml]: I know this was an old issue, but the spelling is actually `operator`

File: app/views/waste_exemptions_engine/on_a_farm_forms/new.html.erb (hunk 1 above)
Comment (eminnett) [reply app/views/waste_exemptions_engine/on_a_farm_forms/new.html.erb]: Well caught.

Comment (Cruikshanks) [reply app/views/waste_exemptions_engine/on_a_farm_forms/new.html.erb]: 🤦‍♂️

File: config/locales/forms/check_your_answers_forms/en.yml (hunk 2 above)
Comment (Cruikshanks) [reply config/locales/forms/check_your_answers_forms/en.yml]: 🤦‍♂️

**Comment by eminnett** [comment]
I can't see anything other than what Iris has already commented on. It's possible I might find something when I write the tests, but I'll fix it then if I do.

**Pull request merged**